### DISCUSSION TITLE
add menu to change screen color / Test Mode

### DISCUSSION THREAD

**Discussion by benja049** [bangcorrupt/hacktribe#109] [Ideas]
it could be cool for customization....

 even if it doesn't have much use these fun 

I use google translation

good job

**Comment by bangcorrupt** [comment]
One time when I crashed it the screen turned red.  It looked cool but was completely illegible.  I haven't looked into how it happened, or if it can be controlled in software.

**Comment by benja049** [comment]
Yes, I have already noticed the blue green and red color...

This happens when doing the startup procedure for the button and screen test.

So, I thought it might be possible to create à menu to change it.


https://user-images.githubusercontent.com/98366332/151098945-41b45420-cef5-4892-91a3-d136c30291ec.mp4

**Reply by bangcorrupt** [reply]
How do you trigger the screen test?

**Reply by benja049** [reply]
Mfx send + 1 + power = test mod

**Reply by bangcorrupt** [reply]
Awesome, thank you.  I saw the menu in the binary but couldn't find how to trigger it.  It looks like the pads are continuously pressure sensitive, so polyphonic after-touch may be possible  some day.

**Reply by bangcorrupt** [reply]
> I thought it might be possible to create à menu to change it.

I think the hardest part here is creating a custom menu.  I haven't made any progress with the UI yet, perhaps extending Step Edit is a way forward.

**Comment by benja049** [comment]
Mfx send + 1 + power = test mod
![20220126_132814](https://user-images.githubusercontent.com/98366332/151163309-01d93485-f106-4bb1-ad95-ce866273a49d.jpg)

Mxf hold + power = menu calibration touche pad

**Reply by bangcorrupt** [reply]
MFX Send + 2 + Power = [SERV] Hardware test
Pattern Set + 1 + Power = [ALL] test
Pattern Set + 2 + Power = [SHIP] test
Pattern Set + 3 + Power = [MAIN] test
Pattern Set + 4 + Power = [PNL] test
Pattern Set + Step Jump + Power = XY Test
Pattern Set + MFX Send + Power = Pad Pressure measurement
